notebooks/236-raft-optimization/236-raft-optimization.ipynb

```diff
@@ -0,0 +1,570 @@
+    "\n",
+    "Here we will convert the raft_kitti_iter20_480x640 model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "sys.path.append(\"../utils\")\n",
+    "import notebook_utils as utils\n",
+    "import tarfile\n",
+    "from pathlib import Path\n",
+    "\n",
+    "base_model_dir = Path(\"models\")\n",
+    "\n",
+    "model_name = \"iter20\"\n",
+    "\n",
+    "archive_name = \"resources.tar.gz\"\n",
+    "\n",
+    "model_url = f\"https://s3.ap-northeast-2.wasabisys.com/pinto-model-zoo/252_RAFT/{model_name}/{archive_name}\"\n",
+    "\n",
+    "downloaded_model_path = base_model_dir / Path(archive_name)\n",
+    "\n",
+    "if not downloaded_model_path.exists():\n",
+    "    utils.download_file(model_url, downloaded_model_path.name, downloaded_model_path.parent)\n",
+    "\n",
+    "model_onnx_path= base_model_dir / Path('raft_kitti_iter20_480x640.onnx')\n",
+    "\n",
+    "if not model_onnx_path.exists():\n",
+    "    with tarfile.open(downloaded_model_path) as file:\n",
+    "        file.extractall(base_model_dir)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Convert model to OpenVINO IR."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "OpenVINO IR already converted\n"
+     ]
+    }
+   ],
+   "source": [
+    "import subprocess\n",
+    "\n",
+    "model_ov_path = model_onnx_path.with_suffix(\".xml\")\n",
+    "\n",
+    "if not model_ov_path.exists():\n",
+    "    # Run Model Optimizer to convert ONNX model to OpenVINO IR.\n",
+    "    mo_command = f'mo --framework onnx -m {model_onnx_path} --output_dir {base_model_dir}'\n",
+    "    subprocess.call(mo_command, shell=True) \n",
+    "else: print('OpenVINO IR already converted')"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Load the model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "input precision: <Type: 'float32'>\n",
+      "input shape: [1,3,480,640]\n",
+      "output precision: <Type: 'float32'>\n",
+      "output shape: [1,2,480,640]\n"
+     ]
+    }
+   ],
+   "source": [
+    "from openvino.runtime import Core\n",
+    "\n",
+    "core = Core()\n",
+    "core.set_property({'CACHE_DIR': '.cache'})\n",
+    "\n",
+    "model_ov = core.read_model(model=model_ov_path)\n",
+    "compiled_model = core.compile_model(model=model_ov, device_name=\"CPU\")\n",
+    "input_layer = compiled_model.input(0)\n",
+    "output_layer = compiled_model.output(1)\n",
```

Comment:
it looks slightly strange that you show only second output from model, could you please provide description why it is need and what imodel predict means. 
Also as I can see below model has 2 inputs, probably you need to show info for both inputs